src/main/java/com/pgs/booking/model/dto/CreateUpdateFlightDto.java

```diff
@@ -0,0 +1,25 @@
+package com.pgs.booking.model.dto;
+
+import lombok.Builder;
+import lombok.Value;
+
+import javax.validation.constraints.NotBlank;
+import java.time.LocalDateTime;
+
+@Value
+@Builder
+public class CreateUpdateFlightDto {
+
+    @NotBlank(message = "Please enter your numberOfPassenger.")
+    String numberOfPassenger;
+    @NotBlank(message = "Please enter your type.")
+    String type;
```

Comment:
a nice touch would be to have it as an enum

Reply:
Ok, to changed.